### PR TITLE
Added SASS solutions, fixed typos, code formatting

### PR DESCRIPTION
**Pull request by drecali**
Added solutions to 3 guide articles. Also fixed typos and added code formatting for one of those articles. I kept the solutions and other edits as separate commits for clarity. Hope this helps. Please let me know if there's anything I can do to improve.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Pleae replace the solution with the full solution, i.r. the seed code with the changes you made for passing. Make it such that if the user copy-pastes the solution you give, the challenge passes

File: guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md [hunk 1]
```diff
@@ -3,8 +3,29 @@ title: Use @if and @else to Add Logic To Your Styles
 ---
 ## Use @if and @else to Add Logic To Your Styles
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+## Solution
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md]: This solution is messed up, requires the HTML part too

File: guide/english/certifications/front-end-libraries/sass/use-for-to-create-a-sass-loop/index.md [hunk 2]
```diff
@@ -50,9 +50,50 @@ The `@for` loop might be useful when combined with CSS’ `:nth-*` pseudo-classe
...
+  }
+  
+  
+</style>
+
+<p class="text-1">Hello</p>
+<p class="text-2">Hello</p>
+<p class="text-3">Hello</p>
+<p class="text-4">Hello</p>
+<p class="text-5">Hello</p>```
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/sass/use-for-to-create-a-sass-loop/index.md]:
```suggestion
<p class="text-5">Hello</p>
```

Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/sass/use-for-to-create-a-sass-loop/index.md]: The three backticks would go on the next line

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by drecali** [comment]
@thecodingaviator thanks! I committed all the suggested changes.

**Review by drecali — COMMENTED** [review]
Finished all the suggested changes. Thanks for the review!

File: guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md [hunk 3]
```diff
@@ -3,8 +3,32 @@ title: Use @if and @else to Add Logic To Your Styles
 ---
 ## Use @if and @else to Add Logic To Your Styles
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+## Solution
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md]: Your given solution resolves to this:
image

Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md]: @Bouncey / @ValeraS Is this a problem on our end?

Comment (RandellDawson) [reply guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md]: @thecodingaviator I copied/pasted the solution provided in this PR and used it in current production and beta site and it passed all tests.  I also tested locally using current master and it passes with no errors.  How were you testing?

Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/sass/use-if-and-else-to-add-logic-to-your-styles/index.md]: I was testing on latest beta, seems fixed now, may have been a problem on my end

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
@drecali Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**